packages/data/src/console.ts

```diff
@@ -0,0 +1,4 @@
+export function error(message: string): never {
```

Comment:
can we put this back in utils? it probably doesn't need to be in a separate file

Reply:
It's only used when creating variants; ts-prune complains it's exported and never used.

Reply:
you mean eslint?

Reply:
oh i see

Reply:
just stick it in createVariants.json then please

Reply:
it's also used in newID.ts

Reply:
can you put it in utils.ts then and add a ts-prune-ignore comment or whatever

Reply:
also i think we should put all of the files related to creating variants in a subdirectory called `createVariants` to stay more organized

Reply:
I put it in createVariants.ts, export it from there and importing it in newID.ts

Reply:
won't that create a dep cycle?
maybe we should create utils.ts in the new createVariants subdirectory

Reply:
All checks pass. Want me to create a new utils.ts?

Reply:
clarification: all checks pass because in this repo there are circular deps, so i disabled that lint rule, but in general we should avoid circular deps in any new code we write

Reply:
since circular deps are spaghetti